### ISSUE TITLE
OpenFOAM dummy path should not be added to rpath/runpath

### ISSUE THREAD

**Issue by mvsurf** [easybuilders/easybuild-framework#3529]
When installing OpenFOAM using rpath, the "dummy" libraries shipped with OpenFOAM are added to the rpath, generating problems with some of the executables, e.g. for exec "snappyHexMesh":

```
--> FOAM FATAL ERROR:
Trying to use the dummy Pstream library.
This dummy library cannot be used in parallel mode

    From function static bool Foam::UPstream::init(int&, char**&, bool)
    in file UPstream.C at line 37.

FOAM exiting
```

which is due by the fact that the exec is linked to:
`$EBROOTOPENFOAM/OpenFOAM-8/platforms/linux64IccDPInt32Opt/lib/dummy/libPstream.so
`

A quick test using the flag "--filter-rpath=dummy" gave the expected results, and the final exec points correctly to the right libPstream.so (`$EBROOTOPENFOAM/OpenFOAM-8/platforms/linux64IccDPInt32Opt/lib/mpi/libPstream.so`)

I was wondering if it could be better to implement this in the toolchain.py (similar of what have been done here: https://github.com/easybuilders/easybuild-framework/pull/2725).

**Comment by boegel** [comment]
@mvsurf Isn't this problem fixed in EasyBuild v4.3.1, see https://github.com/easybuilders/easybuild-easyblocks/pull/2196?

The concept of the "dummy" libraries is very specific to OpenFOAM, I'm not sure we a fix in framework is the best place for that, the OpenFOAM easyblock would be better (in case the PR mentioned above doesn't already fix the issue).

**Comment by mvsurf** [comment]
Sorry for the late reply to this. Indeed I just tested the OpenFOAM easyblock from 4.3.2 and it removes correctly the dummy libraries. Thanks @boegel !
Closing the issue.
